Modules/Segmentation/Voronoi/include/itkVoronoiSegmentationRGBImageFilter.hxx

```diff
@@ -177,32 +177,22 @@ VoronoiSegmentationRGBImageFilter<TInputImage, TOutputImage>::TestHomogeneity(In
     }
   }
 
-  bool ok = true;
   j = 0;
-  double savem, savev;
-  while (ok && (j < 3))
+  while ((j < 3))
```

Comment:
(this is no showstopper, but you may remove one pair of parentheses from `((j < 3))`)

Reply:
Modified to for loop and removed double parenthesis.